lib/requests.js

```diff
@@ -27,14 +29,17 @@ function send(config, options, callback) {
   var url = config.getBaseURL() + path + "?" + queryString;
 
   var params = {
-  	url: url,
-  	proxy: config.proxy,
-  	timeout: config.timeout
+    url: url,
+    proxy: config.proxy,
+    timeout: config.timeout,
+    headers: {
+      'x-pusher-library': 'pusher-http-node ' + packageJson.version
+    }
   };
 
   if(body) {
-  	params.headers = {'content-type': 'application/json'};
-  	params.body = body;
+    params.headers['content-type'] = 'application/json';
+    params.body = body;
```

Comment:
Haha :)

This file at least is 4 spaces everywhere except for those lines which I corrected, which had an errant tab character in them